terra-node-ci/CHANGELOG.md

```diff
@@ -6,4 +6,8 @@
 
 ----------
 
+## 1.0 - (June 3, 2020)
+
```

Comment:
extra lines?

Reply:
the extra lines are there to satisfy the linter that absolutely hates how we format our change logs